torch/linalg/__init__.py

```diff
@@ -378,6 +378,73 @@
             [1, 2, 2, 2]])
 """)
 
+vector_norm = _add_docstr(_linalg.linalg_vector_norm, r"""
+linalg.vector_norm(input, ord=None, keepdim=False, *, out=None, dtype=None) -> Tensor
+
+Returns the vector norm of a given tensor. 
+
```

Comment:
What about a reference to torch.linalg.norm like, "See torch.linalg.norm for computing matrix norms"?